src/convenience_api.jl

```diff
@@ -69,7 +69,7 @@ keyword arguments are applied to the constructed trace.
 function Plot(f::Function, x0::Number, x1::Number, l::Layout=Layout();
               style::Style=CURRENT_STYLE[],
               kwargs...)
-    x = linspace(x0, x1, 50)
+    x = LinRange(x0, x1, 50)
```

Comment:
Shouldn't that just be
```
range(x0, stop = x1, length = 50)
```

Reply:
@Nosferican That is correct too. But does that make a difference? `LinRange` is a type of range.

Reply:
I think Either works, but I prefer the `range` construct as I think it is here to stay.

I'm not sure if LinRange will be considered internal at some point.

@ghuba would you be willing to make that change?

Thanks!

Reply:
@ghuba looks like we posted at the same time.

If this works and is permanent, then I'm ok with with!